content/ADR-202-linked-wearables.md

```diff
@@ -0,0 +1,212 @@
+
+This decision eliminates the need for third parties to implement and maintain API resolvers and utilizes decentralized mechanisms for both content storage and ownership validation.
+
+### LinkedWearable Deployments
+
+When uploading a LinkedWearable to a Catalyst node, the pointer URN and the mapping configuration between the Decentraland wearable and the NFT collection items must be specified.
+
+**Wearables Pointers for Linked Wearables Deployments**:
+The new URN system maintains the existing LinkedWearables pointer when deploying a new linked wearable, allowing seamless migration of old content to the new LinkedWearable system. The previous entity will become inactive and be replaced by the updated version. A critical addition is the mapping of the wearable to its associated NFT collection items, which must be included in the wearable entity during deployment. Several use cases for this mapping are outlined below, with more detailed information available in the [schemas repository](https://github.com/decentraland/schemas/blob/main/src/platform/item/linked-wearable-mappings.ts).
+
+ - **Wearable pointer**: `urn:decentraland:matic:collections-thirdparty:[third-party-name]:[collection-name]:[item-name]`
+   - **thid-party-name**: Name of the third party 
+   - **collection-name**: Name of the Decentraland Wearables collection 
+   - **item-name**: Name of the Wearable 
+
+**Wearables Mappings**: 
+A Decentraland wearable can be linked to one or multiple NFT collection items, meaning that ownership of the specified NFTs grant the user access to the corresponding wearable in Decentraland. The term `mapping` describes the relationship between a Decentraland wearable and its associated NFTs, resulting in several possible use cases described below.
+
+A Decentraland wearable is granted to a user who:
+- Owns a specific NFT collection item
+- Owns any item from the specified NFT collections
+- Owns any item from a list of items within a specified NFT collection
+- Owns any item from a range of items within a specified NFT collection
+
+The mappings configuration supports lists of contracts by chain and multiple sets of collection items, as long as the sets do not overlap with each other in the same collection. This flexibility can accommodate any use case required by a third party. A Decentraland wearable can be linked/mapped to single or multiple items across one or many NFT collections on different chains.
+
+The examples below are mapping configurations within a wearable entity that represent various use cases. This information is sent to the Catalyst during the deployment of the wearable entity. 
+
+- **Owns a specific NFT collection item**: the user owning the item with id `1` in the NFT collection `0x1234567890abcdef1234567890abcdef12345678` on `matic` will be granted with this wearable
+```code:json     
```

Comment:
remove `code:json` cause this block is hidden [here](https://adr.decentraland.org/adr/ADR-202)